packages/confirmation-modal/src/ConfirmationModal.tsx

```diff
@@ -39,19 +50,43 @@ const titleColors = {
 const baseModalStyle = css`
   width: 600px;
   padding: initial;
-`;
-
-const contentStyle = css`
-  font-family: Akzidenz, ‘Helvetica Neue’, Helvetica, Arial, sans-serif;
-  font-size: 14px;
-  line-height: 20px;
   letter-spacing: 0;
-  padding: 36px;
 `;
 
+// TODO: Refresh – remove mode logic
+const contentStyle: Record<Mode, string> = {
+  [Mode.Light]: css`
+    font-family: ${fontFamilies.default};
+    font-size: 13px;
+    line-height: 20px;
+  `,
+  [Mode.Dark]: css`
+    font-family: ${fontFamilies.legacy};
+    font-size: 14px;
+    line-height: 20px;
+    padding: 36px;
+  `,
+};
```

Comment:
Not a blocking change, but what I've started doing is writing the styles the way we want them to be long term here, and then writing the dark mode overrides like what's below. It's ugly, but it'll be easier to remove the temporary darkMode styles later
```
cx(baseStyle,
{
  [css`...`]: darkMode
})
```